hidimstat/knockoffs/utils.py

```diff
@@ -69,6 +71,21 @@ def _bhq_threshold(pvals, fdr=0.1):
     else:
         return -1.0
 
+def _ebh_threshold(evals, fdr=0.1):
```

Comment:
this function should have a test.

Reply:
should i add a test_utils file to the test section? this file is not tested as of now

Reply:
Yes, please.